packages/timeline-state-resolver/src/integrations/vizMSE/__tests__/vizMSE.spec.ts

```diff
@@ -21,8 +21,6 @@ type VRundownMocked = vConnection.VRundownMocked
 import _ = require('underscore')
 import { literal, StatusCode } from '../../../devices/device'
 
-const SHOW_ID = '34ea2db4-ad93-42fb-b352-e49f926ed83f'
```

Comment:
Why not just change the value to `UUID1` instead of removing the constant? You are still parsing a showId in your tests

Reply:
And then you could make another constant for the showName